config/config.exs

```diff
@@ -0,0 +1,12 @@
+use Mix.Config
+
+# config :nerves_firmware_ssh,
+#   authorized_keys: [
+#     "ssh-rsa 12345== abcd",
+#     "ssh-rsa 67890== fghi",
+#   ],
+#   system_dir: "/etc/ssh"
+
+# The keys located in /etc/ssh should be named
+#  ssh_host_rsa_key
```

Comment:
You don't need to change this. I'll get to it when I make a run through the docs, but shouldn't this go in the README.md? I wasn't thinking of having a `config.exs` here since this project is pretty hard to use in isolation.